test/utils/url.spec.ts

```diff
@@ -81,6 +84,131 @@ describe("Url utils tests", () => {
     });
   });
 
+  describe("getTransactionIdAndSubmissionIdFromOriginalUrl tests", () => {
+
+    const urlPathWithIds = "/transaction/123/submission/345";
+    const urlPathWithoutIds = "/path/without/ids";
+    const urlPathWithMissingTransactionId = "/trans/123/submission/345";
+    const urlPathWithMissingSubmissionId = "/transaction/123/sub/345";
```

Comment:
How does the function cope with a URL like "/transaction/submission/no-ids" or perhaps first, what should the function return for this URL?

Reply:
Excellent spot @mwestacott. Have refactored this piece of code to gracefully ignore this type of malformed key/value pairing in the URL. There is a unit test to also check for this too.